tests/factories.py

```diff
@@ -26,7 +26,7 @@ class Meta:
     title = factory.Faker('text', max_nb_chars=75)
     description = factory.Faker('text', max_nb_chars=500)
     status = 'approved'
-    source_id = 'fxsa9'
+    source_id = 'xrfye'
```

Comment:
I've noticed a lot of churn with these guids here. Lets make a ticket to see if theres a way to make this config or make it dynamically.